development/build/scripts.js

```diff
@@ -606,18 +676,23 @@ function setupBundlerDefaults(
     testing,
   },
 ) {
-  const { bundlerOpts } = buildConfiguration;
-
-  Object.assign(bundlerOpts, {
-    // Source transforms
-    transform: [
-      // Remove code that should be excluded from builds of the current type
-      createRemoveFencedCodeTransform(buildType, shouldLintFenceFiles),
-      // Transpile top-level code
-      babelify,
-      // Inline `fs.readFileSync` files
-      brfs,
-    ],
+  const {
+    addDirectOptions,
+    addIgnore,
+    addExclude,
+    addTransform,
+  } = buildConfiguration;
+
+  // Remove code that should be excluded from builds of the current type
+  addTransform(
+    createRemoveFencedCodeTransform(buildType, shouldLintFenceFiles),
+  );
+  // Transpile top-level code
```

Comment:
This is where `tsify` will go. According to the [documentation](https://github.com/TypeStrong/tsify#es2015-formerly-known-as-es6), it needs to go before `babelify`.